test/system/520-checkpoint.bats

```diff
@@ -222,7 +222,7 @@ function teardown() {
     local subnet="$(random_rfc1918_subnet)"
     run_podman network create --subnet "$subnet.0/24" $netname
 
-    run_podman run -d --network $netname $IMAGE sleep inf
+    run_podman run -d --network $netname $IMAGE top
```

Comment:
I changed all `sleep inf`s to `top`. This eliminates the need for `0+w` because `top` is interruptible. 

If there is some important reason for using `sleep`; if checkpoint/restore do not work properly with `top`; please speak now.